### PR TITLE
[netcore] Ongoing work.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

File: mcs/class/System.Private.CoreLib/System.Collections.Generic/Comparer.cs [hunk 1]
```diff
@@ -1,12 +1,45 @@
+using System;
```
Comment (marek-safar) [inline mcs/class/System.Private.CoreLib/System.Collections.Generic/Comparer.cs]: This using is not needed

File: mcs/class/System.Private.CoreLib/System.Collections.Generic/Comparer.cs [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
+
 namespace System.Collections.Generic
 {
+	//[TypeDependencyAttribute("System.Collections.Generic.ObjectComparer`1")]
```
Comment (marek-safar) [inline mcs/class/System.Private.CoreLib/System.Collections.Generic/Comparer.cs]: Why to add this comment?

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**